### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by ScratchAndy13**
Swear in front of me.

### CODE REVIEW

**Comment by ScratchAndy13** [comment]
hi

File: README.md [hunk 1]
```diff
@@ -1 +1,5 @@
-# hello-world
\ No newline at end of file
+# hello-world
+...
+F'ck myself
```
Comment (TrungAnh2020) [inline README.md:3]: What the f*ck?????

File: README.md [hunk 2]
```diff
@@ -1 +1,5 @@
...
+Please
+I want to die
```
Comment (TrungAnh2020) [inline README.md:5]: Really????????????????

**Review by TrungAnh2020 — COMMENTED** [review]
What???????????????????

**Pull request merged**